guides/common/modules/ref_troubleshooting-dns-problems.adoc

```diff
@@ -0,0 +1,27 @@
+:_newdoc-version: 2.18.5
+:_template-generated: 2025-12-02
+:_mod-docs-content-type: REFERENCE
+
+[id="troubleshooting-dns-problems_{context}"]
+= Troubleshooting DNS problems
+
+[role="_abstract"]
+DNS problems might prevent hosts from being provisioned because they cannot correctly resolve the {Project} or {SmartProxyServer} hostnames required to download installation content or configuration files.
```

Comment:
```suggestion
DNS problems might prevent hosts from being provisioned because they cannot correctly resolve {ProjectServer} or {SmartProxyServer} hostnames required to download installation content or configuration files.
```